src/util/apiUtils.js

```diff
@@ -172,6 +172,7 @@ export const buildRequestArgs = externalRequestOpts =>
 			externalRequestOptsQuery.url += `?${dataParams}`;
 			externalRequestOptsQuery.headers['X-Meta-Photo-Host'] = 'secure';
 			break;
+		case 'delete':
```

Comment:
unnecessary - no params need to be sent in a DELETE request to the REST API - it wouldn't know what to do with them anyway.